codes/utils.py

```
@@ -102,4 +102,29 @@ def extract_code_from_content(content):
+
+def to_model_messages(messages_list):
+    """Convert OpenAI-style chat messages to pydantic_ai messages."""
+    from pydantic_ai import messages as ai_messages
+
+    model_messages = []
+    for m in messages_list:
+        role = m.get("role")
+        content = m.get("content", "")
+
+        if role == "system":
+            model_messages.append(ai_messages.ModelRequest(parts=[ai_messages.SystemPromptPart(content)]))
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)